ocpmodels/trainers/forces_trainer.py

```diff
@@ -220,34 +148,6 @@ def load_task(self):
                     )
                     self.normalizers["grad_target"].mean.fill_(0)
 
-        if (
-            self.is_vis
```

Comment:
Makes sense to drop this. Let's also remove the `is_vis` flag from all trainer constructors?